docs/code-of-conduct.md

```diff
@@ -0,0 +1,89 @@
+# Zulip Code of Conduct
+
+## Our Commitment
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers are committed to making participation in our
+project and our community a harassment-free experience for everyone, regardless
+of age, body size, disability, ethnicity, gender identity and expression, level
+of experience, nationality, personal appearance, race, religion, or sexual
+identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being friendly and patient
+* Being respectful of differing viewpoints and experiences, and trying to
+  understand other people's points of view
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+* Assuming good faith on the part of other community members
+* Respecting the time and attention of others, in particular by doing research
```

Comment:
This feels like an admonishment as written and would make me less likely to ask questions if I were a new contributor. I suggest a slightly different approach:

"Supporting each other's learning and participation by using the tools and documentation available and helping to improve them when needed."